<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QWidget" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1013</width>
    <height>529</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>991</width>
    <height>485</height>
   </size>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>test</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <property name="sizeConstraint">
    <enum>QLayout::SetMinimumSize</enum>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="sizeIncrement">
      <size>
       <width>1</width>
       <height>1</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="lineWidth">
      <number>5</number>
     </property>
     <property name="midLineWidth">
      <number>5</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="opaqueResize">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="gridLayoutWidget">
      <layout class="QGridLayout" name="gridLayout" columnminimumwidth="0">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <item row="1" column="0">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="leftMargin">
          <number>5</number>
         </property>
         <property name="topMargin">
          <number>10</number>
         </property>
         <property name="rightMargin">
          <number>5</number>
         </property>
         <property name="bottomMargin">
          <number>12</number>
         </property>
         <item>
          <widget class="QLabel" name="label">
           <property name="font">
            <font>
             <pointsize>15</pointsize>
            </font>
           </property>
           <property name="text">
            <string>SOURCE</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QStackedWidget" name="stackedWidget">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>100</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(221, 221, 221);</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="currentIndex">
            <number>0</number>
           </property>
           <widget class="QWidget" name="page">
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <property name="spacing">
              <number>15</number>
             </property>
             <property name="sizeConstraint">
              <enum>QLayout::SetDefaultConstraint</enum>
             </property>
             <property name="leftMargin">
              <number>20</number>
             </property>
             <property name="topMargin">
              <number>10</number>
             </property>
             <property name="rightMargin">
              <number>20</number>
             </property>
             <property name="bottomMargin">
              <number>10</number>
             </property>
             <item>
              <widget class="QPushButton" name="picButton">
               <property name="font">
                <font>
                 <pointsize>13</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="layoutDirection">
                <enum>Qt::LeftToRight</enum>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton#picButton {
   /*color: rgb(255, 255, 255);*/
   background-color:rgb(255, 255, 255);
/*padding: 10px;*/
padding-left: 5px;
padding-top: 10px;
padding-bottom: 10px;
padding-right: 10px;
color:rgb(70, 70, 70);
}

QPushButton#picButton:hover {
    background-color: rgb(255, 70, 73);
    border-top-left-radius: 8px;
    border-bottom-left-radius: 8px;
/*padding: 10px;*/
padding-left: 5px;
padding-top: 10px;
padding-bottom: 10px;
padding-right: 10px;
color:rgb(70, 70, 70);
}

QPushButton#picButton:pressed {
    background-color: rgb(255, 102, 105);
    border-top-left-radius: 8px;
    border-bottom-left-radius: 8px;
/*padding: 10px;*/
padding-left: 5px;
padding-top: 10px;
padding-bottom: 10px;
padding-right: 10px;
color:rgb(70, 70, 70);
}
</string>
               </property>
               <property name="text">
                <string>    Open cam
    and take 
    a picture     </string>
               </property>
               <property name="icon">
                <iconset>
                 <normalon>../../../../Desktop/photo-camera.png</normalon>
                </iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>52</width>
                 <height>52</height>
                </size>
               </property>
               <property name="flat">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="Line" name="line">
               <property name="frameShadow">
                <enum>QFrame::Sunken</enum>
               </property>
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="vidButton">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="font">
                <font>
                 <pointsize>13</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton#vidButton {
   /*color: rgb(255, 255, 255);*/
   background-color:rgb(255, 255, 255);
/*padding: 10px;*/
padding-left: 5px;
padding-top: 10px;
padding-bottom: 10px;
padding-right: 10px;
	color:rgb(70, 70, 70)
}

QPushButton#vidButton:hover {
    background-color: rgb(255, 70, 73);
    border-top-right-radius: 8px;
    border-bottom-right-radius: 8px;
/*padding: 10px;*/
padding-left: 5px;
padding-top: 10px;
padding-bottom: 10px;
padding-right: 10px;
}

QPushButton#vidButton:pressed {
    background-color: rgb(255, 102, 105);
    border-top-right-radius: 8px;
    border-bottom-right-radius: 8px;
/*padding: 10px;*/
padding-left: 5px;
padding-top: 10px;
padding-bottom: 10px;
padding-right: 10px;
}
</string>
               </property>
               <property name="text">
                <string>    Open cam
    and read 
    from stream</string>
               </property>
               <property name="icon">
                <iconset>
                 <normalon>../../../../Desktop/video-camera (1).png</normalon>
                </iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>60</width>
                 <height>60</height>
                </size>
               </property>
               <property name="flat">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="page_2">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QPushButton" name="pushButton_5">
               <property name="text">
                <string>PushButton</string>
               </property>
               <property name="icon">
                <iconset>
                 <normalon>../../../../Desktop/camera.png</normalon>
                </iconset>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_4">
               <property name="styleSheet">
                <string notr="true">font: 9pt &quot;Isoline&quot;;

border-color: rgb(0, 0, 0);

background-color: rgb(123, 139, 111);

color: rgb(255, 255, 255);
border-top-left-radius: 8px;
border-bottom-left-radius: 8px;
/*border-radius: 10px;
*/
padding: 8;</string>
               </property>
               <property name="text">
                <string>PushButton</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>5</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>5</number>
         </property>
         <property name="bottomMargin">
          <number>5</number>
         </property>
         <item>
          <widget class="QPushButton" name="clearButton">
           <property name="font">
            <font>
             <family>MS Shell Dlg 2</family>
             <pointsize>10</pointsize>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton#clearButton {
   background-color:rgb(194, 194, 194);
padding: 10px;
    border-top-right-radius: 8px;
    border-top-left-radius: 8px;

}

QPushButton#clearButton:hover {
    background-color: rgb(255, 70, 73);
    border-top-right-radius: 8px;
    border-top-left-radius: 8px;
padding: 10px;
}

QPushButton#clearButton:pressed {
    background-color: rgb(255, 102, 105);
    border-top-right-radius: 8px;
    border-top-left-radius: 8px;
padding: 10px;
}
</string>
           </property>
           <property name="text">
            <string>C  L  E  A  R</string>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListWidget" name="picList">
           <property name="font">
            <font>
             <pointsize>15</pointsize>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="sizeAdjustPolicy">
            <enum>QAbstractScrollArea::AdjustToContents</enum>
           </property>
           <property name="dragEnabled">
            <bool>true</bool>
           </property>
           <property name="alternatingRowColors">
            <bool>true</bool>
           </property>
           <property name="resizeMode">
            <enum>QListView::Adjust</enum>
           </property>
           <property name="viewMode">
            <enum>QListView::ListMode</enum>
           </property>
           <property name="modelColumn">
            <number>0</number>
           </property>
           <property name="uniformItemSizes">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <property name="topMargin">
          <number>0</number>
         </property>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget_2">
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="leftMargin">
        <number>5</number>
       </property>
       <property name="rightMargin">
        <number>5</number>
       </property>
       <property name="bottomMargin">
        <number>5</number>
       </property>
       <item row="1" column="0">
        <widget class="QLabel" name="picDisplayer">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>1</width>
           <height>1</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>11</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(221, 221, 221);</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPlainTextEdit" name="plainTextEdit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>70</height>
          </size>
         </property>
         <property name="readOnly">
          <bool>false</bool>
         </property>
         <property name="backgroundVisible">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QWidget" name="topBar" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>10</number>
          </property>
          <property name="bottomMargin">
           <number>1</number>
          </property>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="minimizeButton">
            <property name="styleSheet">
             <string notr="true">QPushButton#minimizeButton {
   /*color: rgb(255, 255, 255);*/
   background-color:rgb(255, 255, 255);
padding: 10px;
}

QPushButton#minimizeButton:hover {
    background-color:rgb(150, 150, 150);
    border-top-left-radius: 8px;
    border-bottom-left-radius: 8px;
padding: 10px;
}

QPushButton#minimizeButton:pressed {
    background-color:rgb(165, 165, 165);
    border-top-left-radius: 8px;
    border-bottom-left-radius: 8px;
padding: 10px;
}
</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normalon>../../../../Desktop/minus-sign.png</normalon>
             </iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>16</width>
              <height>16</height>
             </size>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="closeButton">
            <property name="styleSheet">
             <string notr="true">QPushButton#closeButton {
   /*color: rgb(255, 255, 255);*/
   background-color:rgb(255, 255, 255);
padding: 10px;
}

QPushButton#closeButton:hover {
    background-color: rgb(255, 70, 73);
    border-top-right-radius: 8px;
    border-bottom-right-radius: 8px;
padding: 10px;
}

QPushButton#closeButton:pressed {
    background-color: rgb(255, 102, 105);
    border-top-right-radius: 8px;
    border-bottom-right-radius: 8px;
padding: 10px;
}
</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normalon>../../../../Desktop/cancel (1).png</normalon>
             </iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>16</width>
              <height>16</height>
             </size>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
